tests/README.md

```diff
@@ -4,14 +4,15 @@ Initial test framework for snippets, still a WIP.
 
 # Running
 
-Must have two environment variables set:
-
+Must have one environment variables set:
 
  * `STEPZEN_ENDPOINT` - StepZen endpoint all the tests will be deployed to (one at a time). For example:
    ```
    export STEPZEN_ENDPOINT=https://YOUR_ACCOUNT.stepzen.net/test/snippets/__graphql`
    ```
 
+We use admin key to test because there is a cache optimization for apikey's that is not conducive to rapid deploy and run cycles that occur with this type of testing
```

Comment:
This comment is not relevant here, the README is about running the tests, and nowhere (thanks to this PR) does the user need to set a api key of any type.
More applicable to the code where the admin key is selected.

Reply:
Got it.